osc-server/src/main/java/org/osc/core/broker/service/validator/DeleteDistributedApplianceRequestValidator.java

```diff
@@ -50,6 +57,18 @@ public DistributedAppliance validateAndLoad(BaseDeleteRequest request) throws Ex
                             + request.getId()
                             + " is not marked for deletion and force delete operation is applicable only for entries marked for deletion.");
         }
+        
+        for (VirtualSystem vs : da.getVirtualSystems()) {
+            if (!this.apiFactoryService.supportsNeutronSFC(vs)) {
```

Comment:
we dont care about neutron sfc support. If the vs has a sfc attached to it, we can throw the exception